ppr-ui/src/App.vue

```diff
@@ -134,7 +134,7 @@ export default class App extends Mixins(AuthMixin) {
   }
 
   private get dashboardURL (): string {
-    return window.location.origin + '/dashboard'
+    return window.location.origin + 'dashboard'
```

Comment:
I think localtion.origin will be '.../ppr' so won't this cause an error without the slash?

Reply:
changed the wrong slash